tests/std/tests/P0122R7_span/test.cpp

```diff
@@ -16,6 +16,12 @@
 
 using namespace std;
 
+static_assert(is_standard_layout_v<span<int>>);
+static_assert(is_standard_layout_v<span<int, 4>>);
+
+static_assert(sizeof(span<int, 4>) == sizeof(void*));

```

Comment:
Could we move these down near 90 with an appropriate comment to indicate this is testing a non-standard guarantee? I'd like to keep tests for extensions together in one place.

Reply:
Done